.github/workflows/integration_tests_deployment.yml

```diff
@@ -0,0 +1,68 @@
+name: Integration Tests Deployment
+on:
+  workflow_call:
+    inputs:
+      Environment:
+        type: string
+        required: true
+  workflow_dispatch:
+    inputs:
+      Environment:
+        type: choice
+        description: Choose Environment
+        default: dev
+        options:
+          - dev
+          - prd
+
+env:
+  AWS_REGION: ${{ secrets.AWS_REGION }}
+  ENVIRONMENT: ${{ inputs.Environment }}
+  TEST_ROLE_EXTERNAL_ID: ${{ secrets.TEST_ROLE_EXTERNAL_ID }}
+  PROJECT: domain-protect
+  ROUTE53_ACCOUNT: ${{ secrets.ROUTE53_ACCOUNT }}
+  TEST_ROLE_NAME: ${{ secrets.TEST_ROLE_NAME }}
+  ZONE_NAME: ${{ secrets.ZONE_NAME }}
+  ZONE_ID: ${{ secrets.ZONE_ID }}
```

Comment:
Couldn't the zone id be retrieved from the zone name?

Reply:
Good idea, implemented